orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/token.xml.j2

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<topology>
+   <uri>https://{{ salt['pillar.get']('gateway:address') }}:8443/{{ salt['pillar.get']('gateway:path') }}/token</uri>
+   <name>cdp-token</name>
+   <gateway>
+      <provider>
+         <role>federation</role>
+         <name>SSOCookieProvider</name>
+         <enabled>true</enabled>
+         <param>
+            <name>sso.authentication.provider.url</name>
+            <value>https://{{ salt['pillar.get']('gateway:address') }}:8443/{{ salt['pillar.get']('gateway:path') }}/sso/api/v1/websso</value>
```

Comment:
Is not `/knoxsso/api/v1/websso` the correct path (instead of `/sso/api/v1/websso`)?

Reply:
Probably it is. I couldn't verify this token, just copied over from Ambari code path. I'll fix this.